addons/udes_suggest_location/README.md

```diff
@@ -12,12 +12,14 @@ Module to handle suggested locations
 
 Policies are based on a metaclass that enforces the methods needed for suggested locations to work. Essentially they are used to govern how stock is moved within a warehouse, along with the u_drop_location_constaint.
 
-| Policy | Description |
-| - | - |
-| ByEmpty | Suggest empty locations |
-| ExactlyMatchMoveLine | Suggest the destination that matches the move line's destination location |
-| ByProduct | Match drop locations to locations which are already store the product being dropped off |
-| ByOrigin  | Match drop locations to the origin of the picking |
+| Policy                      | Description                                                                                                                                                                                                                                                                               |
+|-----------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| ByEmpty                     | Suggest empty locations                                                                                                                                                                                                                                                                   |
+| ExactlyMatchMoveLine        | Suggest the destination that matches the move line's destination location                                                                                                                                                                                                                 |
+| ByProduct                   | Match drop locations to locations which are already store the product being dropped off                                                                                                                                                                                                   |
+| ByOrigin                    | Match drop locations to the origin of the picking                                                                                                                                                                                                                                         |
+| ByProductCategory           | Match drop locations to locations which are already store the product being dropped off and order point locations of the product for locations that are linked with product category                                                                                                      |
```

Comment:
Nit: We don't need the "are" here i.e. "locations which already store the product"

Reply:
Should be appreciated that i didnt forget to add in the Readme :)

Reply:
Some one hasnt fill well in the ByProduct policy "Match drop locations to locations which are already store the product being dropped off "